### ISSUE TITLE
NPE in KubernetesResourceUtil.sortEventListBasedOnTimestamp caused by event's lastTimestamp being null

### ISSUE THREAD

**Issue by simkam** [fabric8io/kubernetes-client#3027]
with version 4.13.0
```
java.lang.NullPointerException: text
	at java.util.Objects.requireNonNull(Objects.java:228)
	at java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1848)
	at java.time.Instant.parse(Instant.java:395)
	at io.fabric8.kubernetes.client.utils.KubernetesResourceUtil.lambda$sortEventListBasedOnTimestamp$0(KubernetesResourceUtil.java:330)
	at java.util.TimSort.countRunAndMakeAscending(TimSort.java:355)
	at java.util.TimSort.sort(TimSort.java:234)
	at java.util.Arrays.sort(Arrays.java:1512)
	at java.util.ArrayList.sort(ArrayList.java:1464)
	at io.fabric8.kubernetes.client.utils.KubernetesResourceUtil.sortEventListBasedOnTimestamp(KubernetesResourceUtil.java:328)
```

According to https://github.com/kubernetes/kubernetes/issues/90482, the `firstTimestamp`/`lastTimestamp` and `eventTime` fields are optional, so clients must tolerate events without those timestamps set.

**Comment by rohanKanojia** [comment]
@simkam : Would it be possible for you to raise a PR to fix this?

**Comment by simkam** [comment]
Hi @rohanKanojia, I sent PR with fix. I would like have the fix backported also to `4.13.x` branch? Should I open another PR for `4.13.x`?

**Comment by manusa** [comment]
> Hi @rohanKanojia, I sent PR with fix. I would like have the fix backported also to `4.13.x` branch? Should I open another PR for `4.13.x`?

Your commit was cherry-picked. Release process for 4.13.3 started a few minutes ago.

**Comment by rohanKanojia** [comment]
@simkam :  4.13.3 has been released https://repo1.maven.org/maven2/io/fabric8/kubernetes-client/4.13.3/ . You should be able to find your issue fixed
